src/cpp/lpnamer/input_reader/AdditionalConstraintsReader.cpp

```diff
@@ -37,15 +37,15 @@ void AdditionalConstraintsReader::processSectionLine()
 {
     if ( _line[_line.length()-1] != ']' )
     {
-        std::cout << "line " << _lineNb << " : section line not ending with ']'.\n";
+        std::cout << "additional constraints file:  line " << _lineNb << " : section line not ending with ']'.\n";
```

Comment:
how about to remind the additional constraints file name --> "in additional constraints file:" _path_to_file<<" line "...